tests/FileFormat/PdfStatisticsTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Tests\CubeTools\CubeCommonBundle\FileFormat;
+
+use CubeTools\CubeCommonBundle\FileFormat\PdfStatistics;
+use PHPUnit\Framework\TestCase;
+
+class PdfStatisticsTest extends TestCase
+{
+    protected $folderWithPdfsForTests = 'tests' . DIRECTORY_SEPARATOR . 'Resources' . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR;
+
+    /**
+     * @var \CubeTools\CubeCommonBundle\FileFormat\PdfStatistics
+     */
+    protected $testObject;
+
+    public function setUp()
+    {
+        $this->testObject = new PdfStatistics();
+    }
+
+    public function testGetFormatOfPages()
+    {
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'oneA4grayscale.pdf');
+        $this->assertEquals(array('A4'), $this->testObject->getFormatOfPages());
+
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'twoA4firstGrayscaleSecondColor.pdf');
+        $this->assertEquals(array('A4', 'A4'), $this->testObject->getFormatOfPages());
```

Comment:
Note: One page in another size would have been more interesting. But lets keep it as is.

Reply:
It was more to test if iteration works correctly and if more than one pdf analysed per execution works correctly.